platooning_control/src/platoon_control.cpp

```diff
@@ -49,10 +49,13 @@ namespace platoon_control
         pnh_->param<double>("lowpassGain", config.lowpassGain, config.lowpassGain);
         pnh_->param<double>("lookaheadRatio", config.lookaheadRatio, config.lookaheadRatio);
         pnh_->param<double>("minLookaheadDist", config.minLookaheadDist, config.minLookaheadDist);
+        pnh_->param<double>("wheelBase", config.wheelBase, config.wheelBase);
```

Comment:
@saina-ramyar I'm curious as to why this param is moved from pnh_->getParam() to this? How are the two access methods different and why only this one changed?

Reply:
The way it was before, it was using the vehicle's wheelbase value. Now, it is reading the wheelbase value from the plugin's config. I did it so I can see the impact of wheelbase value on the lane-keeping performance.